osu.Game/Rulesets/Edit/HitObjectComposer.cs

```diff
@@ -383,7 +385,7 @@ public override SnapResult SnapScreenSpacePositionToValidTime(Vector2 screenSpac
 
         public override float GetBeatSnapDistanceAt(HitObject referenceObject)
         {
-            return (float)(100 * EditorBeatmap.Difficulty.SliderMultiplier * referenceObject.DifficultyControlPoint.SliderVelocity / BeatSnapProvider.BeatDivisor);
+            return (float)(100 * referenceObject.DifficultyControlPoint.SliderVelocity * EditorBeatmap.Difficulty.SliderMultiplier / BeatSnapProvider.BeatDivisor);
```

Comment:
What is this change even? It's just a change in the multiplication order that's wedged in here for no discernible reason...?

Reply:
Probably leftover from the beginnings of the PR, can revert I guess.